packages/ui/components/Button.tsx

```diff
@@ -1,89 +1,137 @@
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import styled, { css } from "styled-components";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled, { css } from 'styled-components';
 import {
   ButtonProps,
-  StyledButtonProps
-} from "../../types/src/styled-components/button.types";
+  StyledButtonProps,
+} from '../../types/src/styled-components/button.types';
 
 const StyledButton = styled.button<StyledButtonProps>`
   display: inline-flex;
   flex-direction: column;
-  gap: 8px;
+  gap: 0.5rem;
   align-items: center;
   border-radius: 15px;
-  padding: 9px 14px;
-  font-family: "Poppins";
+  padding: 0.55rem 0.86rem;
+  font-family: 'Poppins', sans-serif;
   font-weight: 500;
   cursor: pointer;
+  transition: 0.2s;
+
   ${({ variant, active, danger, disabled, size }) =>
     css`
-      background-color: ${disabled === true &&
-        variant === "primary" ||
-        variant === "dim"
-        ? "var(--btn-disabled)"
-        : danger === true ||
-        variant === "icon" ||
-        variant === "navIcon"
-        ? "transparent"
-        : variant === "primary"
-        ? "var(--btn-primary)"
-        : variant === "dim"
-        ? "var(--btn-dim)"
-        : "transparent"};
-      border: ${variant === "icon"
-        ? "none"
+      &:hover,
```

Comment:
Have you tested the states? I can't see any color changes.

<img width="1607" alt="Screenshot 2023-03-22 at 00 34 55" src="https://user-images.githubusercontent.com/101993818/226765226-28bf62b8-e6cc-4745-a3d4-02c62fb32818.png">